include/aie/Dialect/AIE/IR/AIEOps.td

```diff
@@ -1650,15 +1650,15 @@ def AIE_ObjectFifoCreateOp: AIE_Op<"objectfifo", [HasParent<"DeviceOp">, Symbol]
 
     The following example shows an objectFifo which transposes a 16x16 matrix of
     `i32`s on the producer side using strides and sizes. No transformation is
-    applied on the consumer side of `%tile13` (in this case the `fromStream`
+    applied on the consumer side of `%tile13` (in this case the `dimensionsFromStreamPerConsumer`
     attribute may also be left off), and a transformation on `%tile23` first gives
```

Comment:
@abisca It seems like the example below doesn't match the description (e.g., fromStream is not left off) or maybe I'm reading it wrong. Can you help correct me in understanding what this examples is showing and how to describe it?

Reply:
The example shows that there is no data layout transformation on the consumer %tile13, you can see that the fromStream attribute there is `[]`. This is different for consumer %tile23 which has a data layout transformation expressed as `[<2, 1>, <128, 2>]`

Reply:
Thank you, that was my mistake to read it incorrectly!